src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -192,6 +192,7 @@ void regclass_Core(py::module m) {
             ov::Tensor tensor(ov::element::Type_t::u8, {});
             return self.read_model(model, tensor);
         },
+        py::keep_alive<0, 1>(),
```

Comment:
Maybe it would be good to add some comment with issue_id tracking fix on c++ side

Reply:
CI is green, i will add a note to revert these python changes in corresponding ticket for c++